src/agent/debuglet.ts

```diff
@@ -291,6 +291,19 @@ export class Debuglet extends EventEmitter {
     const that = this;
     const stat = promisify(fs.stat);
 
+    const workingDir = that.config.workingDirectory;
+    // Don't continue if the working directory is a root directory
+    if (path.join(workingDir, '..') === workingDir) {
+      that.logger.error(
+          `Refusing to start with \`workingDirectory\` set to a root directory:  '${
```

Comment:
Why are there two slightly different error strings for the logger and event emission? Can they be the same?

Reply:
I was trying to make the error message a little more concise than the log message, but I can have them the same.

Reply:
I've updated the error message and log message to be the same.